keyboards/handwired/hillside/52/keymaps/json2hill52.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
```

Comment:
I'm not really a fan of adding python scripts like this.

Reply:
The output from config.fm will come with each layer all on one line and very unreadable. Is there a utility within QMK to pretty print that .json so they can easily maintain the .json keymap by hand or by config.fm?

Reply:
Not one line. One line per key. Also impossible to visualize.

Reply:
I moved the pretty-print script to the Hillside repo wiki, and included a read before running note in both places. I also generalized it more. 

Ideally it should read the keyboard's .h file LAYOUT comments to know the keys per line. But that is a step past the keyboard level, and involves reading the users disk, which I'd rather not do outside the code.